### PR TITLE
Allow existing nc-root element

### PR DESCRIPTION
**Pull request by talves**
**- Summary**

Allow for the render of an existing root element for the CMS app.

### Solves:
Will allow a React app to be able to wrap the CMS component with a react component to be able to include as a route.
[Example Test Repo based on create-react-app](https://github.com/talves/react-netlify-cms)

**- Test plan**

- `yarn build`
- use the `dist` assets in your app

**- Description for the changelog**

Allow existing root element `nc-root` 

**- A picture of a cute animal (not mandatory but encouraged)**
🦈

### CODE REVIEW

**Comment by verythorough** [comment]
Deploy preview for *netlify-cms-www* ready!

Built with commit 354762063c89870924fb00e47ab3e07a636e7a71

https://deploy-preview-1176--netlify-cms-www.netlify.com

**Comment by verythorough** [comment]
Deploy preview for *cms-demo* ready!

Built with commit 354762063c89870924fb00e47ab3e07a636e7a71

https://deploy-preview-1176--cms-demo.netlify.com

File: src/bootstrap.js [hunk 1]
```diff
@@ -25,9 +25,12 @@ function bootstrap(opts = {}) {
   /**
    * Create mount element dynamically.
    */
-  const el = document.createElement('div');
-  el.id = 'nc-root';
-  document.body.appendChild(el);
+  const el = document.getElementById('nc-root');
+  if (!el) {
+    document.createElement('div');
```
Comment (tech4him1) [inline src/bootstrap.js]: This needs to be assigned to `el`, so that you can append it below.

Comment (talves) [reply src/bootstrap.js]: Oops, I have that on mine, forgot to push 👍 @tech4him1 good catch

**Comment by erquhart** [comment]
@talves this is handled under #1173

**Pull request closed without merging**